pkg/kubectl/describe.go

```diff
@@ -842,21 +842,25 @@ func describeContainers(label string, containers []api.Container, containerStatu
 		if len(resourceToQoS) > 0 {
 			fmt.Fprintf(out, "    QoS Tier:\n")
 		}
-		for resource, qos := range resourceToQoS {
+		for _, resource := range SortedQoSResourceNames(resourceToQoS) {
```

Comment:
I think we should stop reporting qos per resource in 1.3

@bgrant0607 @vishh - now that we have merged #14943 should we update the CLI to just output the QoS of the pod?